src/mudlet.cpp

```diff
@@ -1729,6 +1742,7 @@ void mudlet::disableToolbarButtons()
     mpMainToolBar->actions()[14]->setEnabled(false);
 
     mpActionReplay->setEnabled(false);
+    mpActionIRC->setEnabled(false);
```

Comment:
Is this the best place for this line of code - IMHO it would be better placed before or after the pair of lines it is sat between which both act on `mpActionReplay` rather than splitting them...?

Reply:
Sure, applied.